MAINTAINERS-example

```diff
@@ -1,33 +1,23 @@
-# This project, topsikrit, has top-secret awesome sauce.
-#
-# Btw, note that this comment was human-edited and will be preserved
-# even if you add/remove maintainers using the gem. Nifty!
-
-maintainers: {
-    version: 1
-    issues: "https://github.com/csprojects/topsikrit/issues"
-    # This is a very interesting comment. It will also be preserved.
-    roles: {
-        core: {
-            "ada": {},
-            "turing": {},
-        },
-        subsystem: {
-            "gracehopper": { subsystem: "ENIAC" },
-        },
+{
+  "version": 1,
+  "maintained": false,
+  "people": {
+    "gracehopper": {
+      "github": "gracehopper",
+      "email": "admiral.grace.hopper@usnavy.gov",
+      "name": "Grace Hopper",
+      "comment": "Maintains ENIAC subsystem"
```

Comment:
I was thinking more about comments at a higher level than the person, but I don't think it matters much. ❓